addons/grenades/functions/fnc_throwGrenade.sqf

```diff
@@ -56,7 +56,7 @@ if (getNumber (_config >> QGVAR(incendiary)) == 1) then {
     private _fuzeTime = getNumber (_config >> "explosionTime");
     private _timeToLive = getNumber (_config >> "timeToLive");
 
-    [FUNC(incendiary), [_projectile, _timeToLive], _fuzeTime] call CBA_fnc_waitAndExecute;
+    [FUNC(incendiary), [_projectile, _timeToLive, side _unit], _fuzeTime] call CBA_fnc_waitAndExecute; // WE WANT THE OBJECTS SIDE HERE!
```

Comment:
`side OBJECT` (the _unit is an OBJECT) as opposed to `side GROUP` (which would be `side group _unit`)

See: https://community.bistudio.com/wiki/side
>Once dead, a unit will be on the civilian side. Query the side of the Group to get a reliable result.
(It leaves out how objects with `setCaptive` report SIDE_FRIENDLY and renegades report SIDE_ENEMY.)

This is not the wanted behaviour most of the time. We usually use `side group _object`. This is an exception though. We mimick the hard coded AI behaviour and that seems to actually use stuff like `SIDE_ENEMY` and `SIDE_FRIENDLY`. If you make yourself a captive, even enemies won't react to your grenades.